src/components/bs5/navbar/navbar.hbs

```diff
@@ -0,0 +1,41 @@
+<!-- QGDS Partial: Primary navigation -->
+
+<!-- Primary Navigation / Desktop -->
+<nav class="navbar navbar-expand-lg" aria-label="Navbar example">
+    <div class="container-xl">
+        <div class="collapse navbar-collapse">
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                <li class="nav-item nav-item-home">
+                    <a class="nav-link" href="{{metadata.target_url}}">
+                        <span class="qld-icon icon-home"></span>
+                        <span class="visually-hidden">Home</span>
+                    </a>
+                </li>
+                {{#each navigation}}
+                    {{#if dropdown_enabled}}
+                        <li class="nav-item dropdown">
+                            <a class="nav-link dropdown-toggle" href="{{target_url}}" id="megaMenuDropdown" role="button"
+                                data-bs-toggle="dropdown" aria-expanded="false">{{title}}</a>
+                            {{#if dropdown_menu}}
+                            <div class="dropdown-menu w-100" aria-labelledby="megaMenuDropdown">
+                                <div class="row">
+                                    {{#each dropdown_menu.navigation}}
+                                    <div class="col-lg-4"> <!-- I need to add styling/rows/cols etc -->
```

Comment:
Having a hard time replicating the default desktop width of the dropdown as per QH:
e.g https://www.quithq.initiatives.qld.gov.au/